kolibri/content/models.py

```diff
@@ -3,6 +3,31 @@
 To access it, please use the public APIs in api.py
 
 The ONLY public object is ContentNode
+
+Developer note: If you modify the schema here, it has implications for the content
+import pipeline, including Kolibri Studio where the imported content databases
+are created.
+
+As such, if models or fields are added or removed, the CONTENT_SCHEMA_VERSION
+value must be incremented, with an additional constant added for the previous
+version. e.g. when the CONTENT_SCHEMA_VERSION is incremented from '2' to '3'
+a new constant VERSION_2 = '2', should be added.
+
+In addition, the new constant should be added to the mappings dict in
+./utils/channel_import.py with an appropriate ChannelImport class associated. If the
+new schema requires inference of the field when it is missing from old databases
+(i.e. it does not have a default value, or cannot be null or blank), then the
+ChannelImport classes must be updated to infer this data from old databases.
```

Comment:
It's confusing to me that the version constants are strings, and not explicitly ordered. We have a `mappings` object which specifies "schema mappings from one version to another", and the "from" side is explicit, but not the "to" side. Or is it the "to" side that's explicit and not the "from" side? It's not clear to me.

In my head, I expected a more simple integer-based versioning scheme where we simply have a `CONTENT_SCHEMA_VERSION` as an integer, and then we can have a mapping of import functions like:

```python
mappings = {
     1: AddVersionNumber,
     5: IntroduceCoachContent,
}
```

where versions that are not reflected are a no-op (my understanding of `ChannelImport`), and versions that are reflected represent the mapping between version `N-1` and `N`.

Reply:
The target schema is implicit (it is the current schema of this current version of Kolibri that is running).

The mappings are therefore only mappings from the identified version to the current schema of the running version of Kolibri.

There was a reason that I used a string, not an integer, but I honestly can't remember. The explicitness may be slightly cumbersome, but I think this is a case where I would prefer to have it be explicit so that we know exactly what different versions of Kolibri can import.